### DISCUSSION TITLE
https://zongzi531.com/syoukai/index.html

### DISCUSSION THREAD

**Discussion by giscus** [zongzi531/zongzi531.github.io#27] [General]
# https://zongzi531.com/syoukai/index.html

大家好，我是粽子。是一个想学日语的前端菜鸟，想记录一些学习路上的点滴，希望可以在巩固自己的同时帮助到需要帮助的同学...

https://zongzi531.com/syoukai/index.html

**Comment by zongzi531** [comment]
`test`
